unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/ChatWidgetHUD/ChatHUDController.cs

```diff
@@ -152,7 +152,7 @@ private ChatEntryModel ChatMessageToChatEntry(ChatMessage message)
 
         if (message.messageType == ChatMessage.Type.PRIVATE)
         {
-            if (message.recipient == ownProfile.userId)
+            if (message.recipient == ownProfile.userId || message.sender.Contains("fake-user"))
```

Comment:
if you want to know if its a channel you can do: `chatController.GetChannel(message.recipient) != null`